docs/johannes_mueller/anaconda_getting_started/Readme.md

```diff
@@ -1,6 +1,5 @@
 # Getting started with Anaconda and Python
-
```

Comment:
You could maybe also add a link to this blogpost, with a link to your new entry and a little statement that, while Anaconda is still ok to use, we now recommend to switch to Mambaforge because of xyz.

Reply:
Anaconda is broken at the moment. We hit this issue during the last course: https://github.com/conda/conda/issues/11503